### PR TITLE
fix: shellcheck fixes for roxctl.sh

### PR DESCRIPTION
**Pull request by kadern0**
Signed-off-by: Pablo Caderno <kaderno@gmail.com>

## Description

Fix shellcheck warnings for roxctl.sh as per #3544

### CODE REVIEW

**Review by janisz — CHANGES_REQUESTED** [review]
Please remove `dev-tools/roxctl.sh` from https://github.com/stackrox/stackrox/blob/master/scripts/style/shellcheck_skip.txt#L16

File: dev-tools/roxctl.sh [hunk 1]
```diff
@@ -7,13 +7,16 @@ set -eo pipefail
 
 is_operator_on_openshift() {
   local result=0
-  kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null
-  if [[ "$?" -ne "0" ]]; then
+  cmd() {
+      kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null
```
Comment (janisz) [inline dev-tools/roxctl.sh:11]: If we have a function we can check both conditions at once
```suggestion
      kubectl get clusterversions.config.openshift.io version | grep -v "No resources found" > /dev/null && kubectl get centrals.platform.stackrox.io -n stackrox | grep -v "No resources found" > /dev/null
```

**Review by janisz — CHANGES_REQUESTED** [review]

File: dev-tools/roxctl.sh [hunk 2]
```diff
@@ -7,13 +7,16 @@ set -eo pipefail
...
   fi
-
-  kubectl get centrals.platform.stackrox.io -n stackrox | grep -v "No resources found" > /dev/null
-  if [[ "$?" -ne "0" ]]; then
+  mycmd() {
+      kubectl get centrals.platform.stackrox.io -n stackrox | grep -v "No resources found" > /dev/null      
+  }
+  if ! mycmd; then
     result=1
   fi
```
Comment (janisz) [inline dev-tools/roxctl.sh:21]:
```suggestion
```
`mycmd could be removed once you included it in `cmd`

**Comment by porridge** [comment]
Closing this stale PR.
@kadern0 please reopen once you rebase and resolve conflicts.

**Pull request closed without merging**